nuxeo-coldstorage-web/nuxeo-coldstorage.html

```diff
@@ -20,10 +20,19 @@
   </template>
 </nuxeo-slot-content>
 
+<nuxeo-slot-content name="moveToColdStorageDocumentContentAction" slot="DOCUMENT_ACTIONS" order="121">
```

Comment:
Name of action does not follow the same pattern as `moveSelectionContentToColdStorageAction`